src/Persistence/Initialization/VersionSeasonSix/ChaosMixes.cs

```diff
@@ -653,8 +644,6 @@ private ItemCrafting FruitCrafting()
         // Result:
         var fruit = this.Context.CreateNew<ItemCraftingResultItem>();
         fruit.ItemDefinition = this.GameConfiguration.Items.First(i => i.Name == "Fruits");
-        fruit.RandomMinimumLevel = 0;
-        fruit.RandomMaximumLevel = 4;
```

Comment:
Moved this logic to [SimpleItemCraftingHandler.cs](https://github.com/MUnique/OpenMU/pull/543/files#r1875067014)